tests/foreman/ui/test_computeresource_vmware.py

```diff
@@ -593,13 +588,13 @@ def test_positive_provision_end_to_end(
 
     :BZ: 2025523
 
-    :Verifies: SAT-24780
+    :Verifies: SAT-24780, SAT-25810
 
     :customerscenario: true
     """
     SELECTED_ROLE = 'theforeman.foreman_scap_client'
     host_name = gen_string('alpha').lower()
-    guest_os_names = 'Red Hat Enterprise Linux 8 (64 bit)'
+    guest_os_names = 'Red Hat Enterprise Linux 9 (64 bit)'
```

Comment:
Is it on purpose that the RHEL version here is 9, and on line [560](https://github.com/SatelliteQE/robottelo/blob/26fec4d0cbbcef3cb6f123f0de075153b61707a4/tests/foreman/ui/test_computeresource_vmware.py#L560) we are matching it to 8?

Reply:
Good catch, I've updated